dashboard/app/controllers/application_controller.rb

```diff
@@ -70,6 +70,12 @@ def render_500
     end
   end
 
+  def prevent_caching
```

Comment:
looks like we can now remove prevent_caching from ScriptLevelsController:

https://github.com/code-dot-org/code-dot-org/blob/3ac4fe0ea8befdfd0ae59fe3988acee06c6dc202/dashboard/app/controllers/script_levels_controller.rb#L7